frontend/src/pages/RequestorPage/RequestPanel/RequestPanel.tsx

```diff
@@ -79,7 +79,7 @@ function ResizableRequestMeta(props: RequestPanelProps) {
   const styleWidth = useStyleWidth(width);
   return (
     <Resizable
-      className="min-w-[200px] overflow-hidden h-full"
```

Comment:
without the `overflow-hidden` here we don't get scrollability for the inner contents when they overflow (many request params, long req body)

Reply:
then once we do have a scrollable container, might need to add some bottom padding: 
<img width="335" alt="image" src="https://github.com/user-attachments/assets/36ba0755-5e2f-487f-afd4-19a2d8252f26">